src/state_machine.zig

```diff
@@ -380,6 +426,10 @@ pub fn StateMachineType(
                 .lookup_transfers => {
                     self.prefetch_lookup_transfers(mem.bytesAsSlice(u128, input));
                 },
+                .get_account_transfers => {
+                    const slice = mem.bytesAsSlice(GetAccountTransfers, input);
```

Comment:
We can't `assert` here, it would panic the server if the client sent an incorrect message.
Let's check the `input` size and return an empty result in this case.

Reply:
Empty result is a valid result to `get_account_transfers`, though. :thinking: